internal/v2/repository/org/org.go

```diff
@@ -174,3 +174,38 @@ func OrgReactivatedEventMapper(event *repository.Event) (eventstore.EventReader,
 
 	return orgChanged, nil
 }
+
+type OrgRemovedEvent struct {
+	eventstore.BaseEvent `json:"-"`
+	Name                 string
```

Comment:
should be private as it's only used for unique constraint
and there's no data, ...